lib/cdk-project-stack.ts

```diff
@@ -35,6 +36,70 @@ export class CdkProjectStack extends cdk.Stack {
     });
     const apiIntegration = new apigateway.LambdaIntegration(lambdaFunction);
 
+    //Cognito
+    const userPool = new cognito.UserPool(this, 'userPool', {
+      selfSignUpEnabled: true,
+      userVerification: {
+        emailSubject: 'Verify your email!',
+        emailBody: 'Hello, Thanks for signing up! {##Verify Email##}',
+        emailStyle: cognito.VerificationEmailStyle.LINK
+      },
+      signInAliases: {
+        username: true,
+        email: true
+      },
+      standardAttributes: {
+        email: {
+          required: true
+        }
+      }
+    });
+
+    const userPoolIdentityProviderGoogle = new cognito.UserPoolIdentityProviderGoogle(this, 'userPoolGoogle', {
+      userPool: userPool,
+      //OAuth 2.0 Client ID/Secret generated in my GCP account
+      clientId: '220086216015-vq3ai6fr1qutbsc40aafk4b31mg1s7gm.apps.googleusercontent.com',
+      clientSecret: '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',
```

Comment:
remove this, and reset the secret on your cognito user pool. My recommendation is to setup the secret in Secrets Manager manually and pass that secret in here when you build - https://bobbyhadz.com/blog/get-secrets-manager-values-aws-cdk